Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -74,6 +74,25 @@
   - type: Appearance
   - type: WiresVisuals
 
+- type: entity
+  parent: VendingMachine
+  id: VendingMachineCondiments
+  name: Condiment Station
+  description: Slather these thick gooey substances on your food for a full flavor effect.
+  components:
+  - type: Sprite
+    sprite: Structures/dispensers.rsi
```

Comment:
This should have its own RSI in the vending machines directory.

The sprites should follow the other vendors more closely. Since it looks like this is powered it should have a power off state and a state for a maintenance panel.

Reply:
I guess I was looking at the dispensers, like booze/soda that didn't have power states. I already have a powered off though. So, not a problem.